src/store/modules/imageUploaderStore.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+const aws = require('aws-sdk')
```

Comment:
You did not include this dependency in the package.json file

Reply:
"dependencies": {
        "autoprefixer": "^6.7.2",
        "aws-sdk": "^2.1231.0",
        "axios": "^0.16.2",